src/Sql/Sql/help/New-AzSqlInstanceLink.md

```diff
@@ -0,0 +1,301 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version:
+schema: 2.0.0
+---
+
+# New-AzSqlInstanceLink
+
+## SYNOPSIS
+Creates a new Azure SQL Managed Instance Link.
+
+## SYNTAX
+
+### CreateByNameParameterSet (Default)
+```
+New-AzSqlInstanceLink [-ResourceGroupName] <String> [-InstanceName] <String> [-LinkName] <String>
+ [-PrimaryAvailabilityGroupName] <String> [-SecondaryAvailabilityGroupName] <String> [-TargetDatabase] <String>
+ [-SourceEndpoint] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### CreateByParentObjectParameterSet
+```
+New-AzSqlInstanceLink [-LinkName] <String> [-PrimaryAvailabilityGroupName] <String>
+ [-SecondaryAvailabilityGroupName] <String> [-TargetDatabase] <String> [-SourceEndpoint] <String>
+ [-Instance] <AzureSqlManagedInstanceModel> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
```

Comment:
Recommend Instance->InstanceObject, please check whether this recommendation is applicable to other cmdlets.

Reply:
It is applicable, thanks!